src/redux/modules/submissions.js

```diff
@@ -136,6 +148,27 @@ const reducer = handleActions(
                 )
               : item
         )
+      ),
+    [additionalActionTypes.UNACCEPT_PENDING]: (state, { meta: { id } }) =>
+      state.setIn(['resources', id, 'data', 'accepted'], false),
+
```

Comment:
I am not sure whether this optimistic update is valid any longer and the same goes for ACCEPT_PENDING. The problem is, what if the user is a click-o-maniac, which clicks twice in a row. Let us say that first click is on the Accept button. The accept operation will be started (in async) and Unaccept button will be displayed. Then the user clicks on Unaccept and start another async operation before the first one gets finished. In theory, the second request can be handled before the first one (at least from the DB TX point of view), which would lead into situation that the Accept state will prevail.